packages/wayland_protocols.rb

```diff
@@ -1,35 +1,20 @@
 require 'package'
 
 class Wayland_protocols < Package
-  description 'Wayland is a protocol for a compositor to talk to its clients'
-  homepage 'https://wayland.freedesktop.org'
-  version '1.13'
-  source_url 'https://wayland.freedesktop.org/releases/wayland-protocols-1.13.tar.xz'
-  source_sha256 '0758bc8008d5332f431b2a84fea7de64d971ce270ed208206a098ff2ebc68f38'
+  description 'Wayland is a protocol for a compositor to talk to its clients.'
+  homepage 'https://wayland.freedesktop.org/'
+  version '1.17'
+  source_url 'https://wayland.freedesktop.org/releases/wayland-protocols-1.17.tar.xz'
+  source_sha256 'df1319cf9705643aea9fd16f9056f4e5b2471bd10c0cc3713d4a4cdc23d6812f'
 
-  binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/wayland_protocols-1.13-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/wayland_protocols-1.13-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/wayland_protocols-1.13-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/wayland_protocols-1.13-chromeos-x86_64.tar.xz',
-
-  })
-  binary_sha256 ({
-    aarch64: 'bd0eb42679be30bddbbe20bb254508be16d8a4f5566c246885fb20a94a2fc3d7',
-     armv7l: 'bd0eb42679be30bddbbe20bb254508be16d8a4f5566c246885fb20a94a2fc3d7',
-       i686: 'ef466cc4ddca3c0d6069c3cdb110d5024b533d78be6512dff377751affe76f94',
-     x86_64: '3d3471f15e2f7e80cef25f272d1f0e29cb0e323e78beaa479df8d319f3b3d477',
-
-  })
-
-  depends_on 'libwayland'
```

Comment:
Because `wayland` already depends on it and it messes up the dependency order.
For example, `wayland_protocols` depends on `wayland-scanner` from `libwayland`. However, if the dependencies are not set up in such a way as they are now, Chromebrew tries to install `wayland_protocols` before `libwayland`.

TL;DR

`libwayland` is already pulled in by `wayland`, so the dependency is redundant.

Reply:
But isn't wayland a fake package?  What if we just removed it?  I want to get away from fake packages if they aren't necessary.  This looks like that might be the case.

Reply:
Would you be comfortable renaming `wayland_protocols` to `wayland`?

Reply:
Change the `depends_on` probably makes the most sense.

Reply:
@JL2210: Were you going to change `depends_on 'wayland'` to `depends_on 'wayland_protocols'` in all the packages?

Reply:
@JL2210:  You nudge others now it's my turn to nudge you. :)

Reply:
I did. There's only ~3 that depend on `wayland`.

Reply:
Sorry. Must have not pushed it.